lib/inferno/dsl/fhir_evaluation/evaluation_context.rb

```diff
@@ -0,0 +1,27 @@
+module Inferno
+  module DSL
+    module FHIREvaluation
+      # EvaluationContext is a wrapper class around the concepts needed to perform an evaluation:
+      # - The IG used as the basis for evaluation
+      # - The data being evaluated
+      # - A summary/characterization of the data
+      # - Evaluation results
+      class EvaluationContext
+        attr_reader :ig, :data, :results, :config
+
+        # rubocop:disable Naming/MethodParameterName
```

Comment:
Instead of using a disable and an enable, just put the disable at the end of the relevant line, and then it will only affect that line and you don't have to reenable the rule.

Reply:
Thanks for the tips. Modified.